pyproject.toml

```diff
@@ -42,4 +40,9 @@ build-backend = "hatchling.build"
 [tool.pytest.ini_options]
 addopts = [
     "--import-mode=importlib",
-]
\ No newline at end of file
+]
+
+[tool.uv]
```

Comment:
Is adding this new tool required for the code changes? Why is adding `uv` a good idea, given the addition of new dependencies?

Reply:
It's not strictly speaking required, but from a development POV I like it quite a bit. It's fast and makes getting started and contributing a lot easier; a dev can run tests or the code without having to figure out installing Python or anything else, just `uv run fediblock-sync ...`

Reply:
It looks useful as part of the CI PR. I basically re-implemented this PR in #67 because I wasn't sure when you'd be able to respond, and wanted to split out the parts that confused me so I didn't block getting the functionality into main. Also the linting changes I did to prepare the ground for this and #64 caused merge conflicts.

Let's put the build/CI/version stuff into the CI PR and I'll merge #67 instead of this one so you don't have to muck about fixing merge conflicts and other minor quibbles of mine.

I've tried to make it clear that it's your work so you get the credit for the PR.